packages/datasource-sql/src/index.ts

```diff
@@ -50,12 +52,26 @@ export async function buildSequelizeInstance(
 
 export function createSqlDataSource(
   uriOrOptions: PlainConnectionOptionsOrUri,
-  options?: { introspection: Table[] },
+  options?: { introspection: Table[] | Introspection },
```

Comment:
```suggestion
  options?: { introspection: Introspection | Introspection.tables },
```
I know that might not sound like a big deal but we have a better idea like this IMO, what do you think?